web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx

```diff
@@ -403,3 +458,11 @@ export default (props: {
     </>
   )
 }
+
+function CreateSimErrorDialog() {
+  return (
+    <Typography color="danger" variant="h6">
+      Something went wrong, try changing to a unique label.
```

Comment:
The API actually returns a useful error message:
"message: "DuplicateFileNameException: 'ForecastDS/simulationConfigs/stooSim' already exists""
Should use a variation of that instead of this more ambiguous "Something...".

Reply:
Great suggestion, fixed!